piksi_tools/fileio.py

```diff
@@ -38,13 +39,14 @@ def read(self, filename):
     buf = ''
     while True:
       msg = struct.pack("<IB", len(buf), chunksize) + filename + '\0'
-      self.link.send(SBP_MSG_FILEIO_READ, msg)
-      data = self.link.wait(SBP_MSG_FILEIO_READ, timeout=1.0)
+      self.link.send(SBP_MSG_FILEIO_READ_REQUEST, msg)
+      data = self.link.wait([SBP_MSG_FILEIO_READ_REQUEST,
```

Comment:
You can wait on multiple messages... one of the benefits that came out of https://github.com/swift-nav/libsbp/pull/174.